tools/util.c

```diff
@@ -584,3 +570,9 @@ plural(size_t x)
 {
 	return x == 1 ? "" : "s";
 }
+
+int
+should_retry_with_pin(int r)
+{
+	return r == FIDO_ERR_PIN_REQUIRED || r == FIDO_ERR_UV_INVALID;
```

Comment:
for authenticators that do not support `clientPin` this will prompt the user for a non-existing pin if built-in UV fails, right?